src/System.Private.ServiceModel/tools/CertificateGenerator/App.config

```diff
@@ -6,6 +6,6 @@
     <add key="CrlFileLocation" value="c:\wcftest\test.crl"/>
   </appSettings>
   <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5"/>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8"/>
   </startup>
```

Comment:
What was the reason for raising the supported runtime version to 4.8?

Reply:
Building the CertificateGenerator tool using VS 2022 Developer Command Prompt requires to install .NET 4.5 SDK/Targeting Pack manually, I thought it would be nice to upgrade the runtime version and build the tool directly. This is what I get when using .NET 4.5:
Error MSB3644: The reference assemblies for .NETFramework,Version=v4.5 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks